### PR TITLE
implement Display for Duration

### LINKED ISSUE #31: impl Display for Duration
Similar to the implementations for `Date`, `Time`, `DateTime`.

### PR DESCRIPTION
**Pull request by hoodie**
should fix #31 
* [x] find a representation for zero duration

### CODE REVIEW

File: src/assert.rs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        assert_eq!(
+            parsed, $expectation,
+            "{:?} not parsed as expected (leftover: {:?})",
+            $line, rest
+        );
+        assert!(rest.is_empty(), "not parsed completely");
+
+        let serialized = parsed.to_string();
+        assert_eq!($line, serialized, "does not reserialize correctly");
+        println!("✅");
```
Comment (badboy) [inline src/assert.rs:27]: Hm, I guess these won't show up anyway unless it fails or we use `--nocapture`, right?
Then I don't mind too much.

File: src/display.rs [hunk 2]
```diff
@@ -37,3 +37,104 @@ impl Display for DateTime {
         write!(f, "{}T{}", self.date, self.time)
     }
 }
+
+impl Display for Duration {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        // P1Y2M3DT4H5M6S
```
Comment (badboy) [inline src/display.rs:43]: Unnecessary comment?

**Review by badboy — APPROVED** [review]

File: src/display.rs (hunk 2 above)
Comment (hoodie) [reply src/display.rs:43]: that was just a mental crutch for me, will remove it

Comment (hoodie) [reply src/display.rs]: removed!

**Pull request merged**